.env

```
@@ -1,12 +1,21 @@
+CLOUDINARY_API_SECRET=n8Nr4bMYGioiE4fssNV2uI6Rcq4
+
+# ✅ Email — Configurar con Resend.com (https://resend.com)
+# 1. Crear cuenta gratis en resend.com
+# 2. Ir a API Keys → crear una clave
+# 3. Reemplazar los valores de abajo:
+EMAIL_HOST=smtp.resend.com
+EMAIL_PORT=465
+EMAIL_SECURE=true
+EMAIL_USER=resend                        # Resend usa "resend" como usuario SMTP
+EMAIL_PASS=re_XXXXXXXXXXXXXXXXXXXXXXXXXX # Reemplazar con tu API key de Resend
+EMAIL_FROM="Eat & Run" <onboarding@resend.dev>  # Cambiar por tu dominio verificado
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_FROM key should go before the EMAIL_HOST key

(UnorderedKey)